incubator/hnc/api/v1alpha1/hnc_config.go

```diff
@@ -156,8 +155,7 @@ type HNCConfigurationCondition struct {
 	//
 	// Currently, the supported values are:
 	//
-	// - "critSingletonNameInvalid": the specified singleton name is invalid. The name should be the
-	// same as HNCConfigSingleton.
+	// (We don't have any critical conditions for now.)
```

Comment:
I still kept the description (line 150-154) about critical or non-critical condition. Please lmk if you would prefer that to be removed entirely.